lib/build.gradle

```diff
@@ -21,6 +22,8 @@ dependencies {
 			configurations.nimbus,
```

Comment:
i think configurations.powertools can be removed from `implementation` ?

Reply:
addressed @tal-nagra , thanks!